src/LaravelBasicSearch.php

```diff
@@ -1,11 +1,18 @@
 <?php
 
 namespace JianJye;
-
 class LaravelBasicSearch
 {
-    public static function search($request, $model, $fields, $ranges = [], $sorts =[])
+    public static function search($request, $model, $fields, $ranges = [], $sorts = [], $dates = [])
     {
+        foreach ($dates as $date => $format) {
+            if (!empty($request->query($date))) {
+                $request->merge([
+                    $date => \Carbon\Carbon::createFromFormat($format, $request->query($date))->format('Y-m-d'),
```

Comment:
Can't we just do like `new \Carbon\Carbon($date)`? That way we can support hour or minute even.